content/using-atom/sections/basic-customization.asciidoc

```diff
@@ -65,12 +65,13 @@ You can open this file in an editor from the _Atom > Open Your Stylesheet_ menu.
 .Open your stylesheet
 image::../../images/menubar.png[open stylesheet]
 
-For example, to change the color of the cursor, you could add the following rule to your _~/.atom/styles.less_ file:
+For example, to change the colors of the status-bar, you could add the following rule to your _~/.atom/styles.less_ file:
```

Comment:
We should probably avoid referring to the path of `~/.atom/styles.less` and just use `styles.less` until we have a method of encoding platform-distinctive documentation.